### ISSUE TITLE
Adding documentation for virtio-mem Windows support on https://virtio-mem.gitlab.io/

### ISSUE THREAD

**Issue by ohault** [virtio-win/kvm-guest-drivers-windows#1161]
For competeness about the avaibility of Windows implementation of virtio-mem, does it possible to update the whole https://virtio-mem.gitlab.io/ ?

I would think about a dedicated User Guide for Windows and/or Hyper-V hypervisor and/or for Cloud Hypervisor

**Comment by YanVugenfirer** [comment]
@kostyanf14

**Comment by kostyanf14** [comment]
@davidhildenbrand can you update this wiki? As I understand from comment  https://github.com/virtio-win/kvm-guest-drivers-windows/pull/794#issuecomment-2074467063 you have access to do this

**Comment by davidhildenbrand** [comment]
@kostyanf14 yes, I maintain that site. I added some high-level virtio-win/Windows details on the [front page](https://virtio-mem.gitlab.io/) just recently.

I asked cloud-hypervisor folks quite a while ago if there is something worth documenting, but never heard back.

Regarding Hyper-V, I don't think they implement virtio-mem; they primarily make use of the Hyper-V balloon ("dynamic memory").

If someone tells me what to document regarding virtio-win, I'm happy to document it. Alternatively, the site is available on github [here](https://gitlab.com/virtio-mem/virtio-mem.gitlab.io), and people should be able to open MRs that I'll happily review+merge :)

**Comment by ohault** [comment]
@davidhildenbrand thank you for this update. Adding a User Guide for Windows here will be appreciated.
https://gitlab.com/virtio-mem/virtio-mem.gitlab.io/-/tree/master/user-guide?ref_type=heads

**Comment by davidhildenbrand** [comment]
@ohault right. Awaiting input from virt-win folks (or even better, a PR :) ).

**Comment by YanVugenfirer** [comment]
This issue has been tracked in Jira: https://redhat.atlassian.net/browse/RHELMISC-34266
